Bookstore: styling

lily-coder opened the pull request:
In this section I have added styling to the page

lily-coder committed:
Add styling

lily-coder committed:
Add project screenshot

afanimartin reviewed (changes requested):
Hi @lily-coder, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Your design doesn't quite look like the given mockup. Please take note of the following sections of the design.
       --> vertically align the avatar, the list of books and `ADD BOOK` button
       --> add chapter titles for every book
       --> the horizontal line should be the same width as the list of books
       --> change the title `ADD A NEW BOOK` to `ADD NEW BOOK` and the title should be vertically aligned with the list of books
       --> the original design only has the book title, category and submit button but you added an extra author's field. Please remove the extra field.

#### Refer to the original design
![main-ui](https://user-images.githubusercontent.com/80149314/136671933-d4313546-dda3-4a33-ab76-86c784c34051.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lily-coder committed:
Update page to match original

lily-coder commented:
I have implemented all changes as requested by reviewer:
- I have vertically aligned the avatar, the list of books and ADD BOOK button.
- I have added chapter titles for every book.
- I have made the horizontal line same width as the list of books.
- I have changed the title ADD A NEW BOOK to ADD NEW BOOK and the title is now vertically aligned with the list of books
- I have removed the extra author's field.

lily-coder committed:
Add project screenshot

addod19 reviewed (approved):
### Hi @lily-coder, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional
- [ ] It would be better if we implement mobile responsiveness of our app, this is a great skill to have, to make websites responsive is a great front end skill


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
